docs/sources/developers/plugins/create-a-grafana-plugin/extend-a-plugin/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Extend a Grafana plugin
+menuTitle: Extend a plugin
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - development
+  - extension
+  - documentation
+description: An index of how-to topics for extending or enhancing Grafana plugins.
+weight: 200
+---
+
+# Extend a Grafana plugin
+
+This section contains how-to topics for extending or enhancing Grafana plugins.
+
+- [Add annotations]({{< relref "./add-support-for-annotations.md" >}})
+- [Add anonymous usage reporting]({{< relref "./add-anonymous-usage-reporting.md" >}})
+- [Add authentication for a data source plugin]({{< relref "./add-authentication-for-data-source-plugins.md" >}})
+- [Add distributed tracing for backend plugins]({{< relref "./add-distributed-tracing-for-backend-plugins.md" >}})
+- [Add Explore queries]({{< relref "./add-support-for-explore-queries.md" >}})
+- [Add query editor help]({{< relref "./add-query-editor-help.md" >}})
+- [Add variables]({{< relref "./add-support-for-variables.md" >}})
+- [Automate development with CI](https://grafana.github.io/plugin-tools/docs/ci)
+- [Build a custom panel option editor]({{< relref "./custom-panel-option-editors.md" >}})
+- [Create nested plugins](https://grafana.github.io/plugin-tools/docs/nested-plugins)
+- [Extend configurations](https://grafana.github.io/plugin-tools/docs/advanced-configuration)
+- [Work with cross-plugin links]({{< relref "./cross-plugin-linking.md" >}})
```

Comment:
I think we'll likely need to bring this and the missing `extend-the-grafana-ui-with-links` into their own section in the near term future, but this is fine for now

Reply:
Got it, let's do that.